crates/modules/smtp-notifier/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+use pulsar_core::event::Threat;
+use pulsar_core::pdk::{
+    CleanExit, ConfigError, ModuleConfig, ModuleContext, ModuleError, PulsarModule, ShutdownSignal,
+    Version,
+};
+
+use lettre::{
+    transport::smtp::authentication::Credentials, AsyncSmtpTransport, AsyncTransport, Message,
+    Tokio1Executor,
+};
+
+const MODULE_NAME: &str = "smtp-notifier";
+
+pub fn module() -> PulsarModule {
+    PulsarModule::new(
+        MODULE_NAME,
+        Version::parse(env!("CARGO_PKG_VERSION")).unwrap(),
+        smtp_notifier_task,
+    )
+}
+
+async fn smtp_notifier_task(
+    ctx: ModuleContext,
+    mut shutdown: ShutdownSignal,
+) -> Result<CleanExit, ModuleError> {
+    let mut receiver = ctx.get_receiver();
+    let mut rx_config = ctx.get_config();
+    let mut config: SmtpNotifierConfig = rx_config.read()?;
+
+    loop {
+        tokio::select! {
+            // Handle configuration changes:
+            _ = rx_config.changed() => {
+                config = rx_config.read()?;
+                continue;
+            }
+            r = shutdown.recv() => return r,
+            msg = receiver.recv() => {
+                let event = msg?;
+
+                // Check if the even is a threat and send a email if it is
+                if let Some(Threat {
+                    source,
+                    description,
+                    extra: _,
+                }) = &event.header().threat
+                {
+                    let payload = event.payload();
+                    let subject = format!("Pulsar Threat Notification: {source}");
+                    let body = format!("{description}\n Source event: {payload}");
+
+                    let message = Message::builder()
+                        .from("Pulsar <pulsar-threat-notification@gmail.com>".parse()?)
+                        .to(config.receiver.parse()?)
+                        .subject(subject)
+                        .body(body)?;
+
+                    AsyncSmtpTransport::<Tokio1Executor>::relay(config.server.as_str())?
+                        .credentials(Credentials::new(config.user.clone(), config.password.clone()))
+                        .build()
+                        .send(message)
+                        .await?;
+                }
+            }
+        }
+    }
+}
+
+#[derive(Clone, Debug, Default)]
+struct SmtpNotifierConfig {
+    server: String,
+    user: String,
+    password: String,
+    receiver: String,
+}
```

Comment:
you should also add `port` in the configuration and an option for the encryption, something like:
```rust
enum Encryption {
     Plain,
     TLS,
     StartTLS
}
```